www/docs/en/dev/guide/overview/index.md

```diff
@@ -45,39 +43,56 @@ Use Apache Cordova if you are:
   device-level APIs, or if you want to develop a plugin interface
   between native and WebView components.
 
-## Basic Components
+# Architecture
 
-Apache Cordova applications rely on a common `config.xml` file that provides
-information about the app and specifies parameters affecting how it
-works, such as whether it responds to orientation shifts. This file
-adheres to the W3C's
-[Packaged Web App](http://www.w3.org/TR/widgets/),
-or _widget_, specification.
+There are several components to your cordova application. The following 
+diagram shows a high-level view of the cordova application architecture. 
+    
+![]({{ site.baseurl }}/static/img/guide/cordovaapparchitecture.png)
 
-The application itself is implemented as a web page, by default a local
-file named _index.html_, that references whatever CSS, JavaScript,
-images, media files, or other resources are necessary for it to run.
-The app executes as a _WebView_ within the native application wrapper,
-which you distribute to app stores.
+## WebView
 
 The Cordova-enabled WebView may provide the application with its
 entire user interface. On some platforms, it can also be a component
 within a larger, hybrid application that mixes the WebView with native
-application components. (See [Embedding WebViews](../hybrid/webviews/index.html) for details.)
-
-A _plugin_ interface is available for Cordova and native components to
-communicate with each other. This enables you to invoke native code
-from JavaScript. Ideally, the JavaScript APIs to that native code are
-consistent across multiple device platforms. As of version 3.0, plugins provide
-bindings to standard device APIs.  Third-party plugins provide
-additional bindings to features not necessarily available on all
-platforms. You can find these third-party plugins in the
-[plugin registry](http://plugins.cordova.io) and use them in your
-application. You can also develop your own plugins, as described in the
-[Plugin Development Guide](../hybrid/plugins/index.html). Plugins may be necessary, for example, to
-communicate between Cordova and custom native components.
-
-__NOTE__: As of version 3.0, when you create a Cordova project it does not have
+application components. 
+(See [Embedding WebViews](../hybrid/webviews/index.html) for details.)
+
+## Web App
+
+This is the part where your application code resides. The application itself is 
+implemented as a web page, by default a local file named _index.html_, that 
+references whatever CSS, JavaScript, images, media files, or other resources 
+are necessary for it to run. The app executes as a _WebView_ within the native 
+application wrapper, which you distribute to app stores.
+
+This container has a very crucial file - `config.xml` file that provides
```

Comment:
Do we need to talk about config.xml in this page? Might be better leaving that to the config.xml topic page.